src/index.ts

```diff
@@ -69,7 +69,7 @@ async function main() {
           type: "confirm",
           name: "proceed",
           message:
-            "Running this command will install dependencies and overwrite your existing tailwind.config.js. Proceed?",
+            "Running this command will install dependencies and. Proceed?",
```

Comment:
There is an extra `and.`

It should be `"Running this command will install dependencies. Proceed?"`